client/src/components/ajax.js

```diff
@@ -0,0 +1,8 @@
+import $ from 'jquery';
+
+const fetchShopItems = (callback) => {
+  $.get('http://localhost:3000/shop', (data) => {
+    callback(null, data);
+  });
```

Comment:
how are you requesting a specific shop's items without sending it a shop id? your endpoint should probably be `/shops/:shopId` at the minimum. If you want to be more in line with REST it would be `/shops/:shopId/items` which is basically saying "give me all the items for the shop with this id"

also, are you thinking about how you will get to a given shop when all you're going to be given is an itemId? (since we will be navigating the page according to an itemId not a shopId). I have the same setup for my component, so I will be querying my DB for the given itemId, then retrieving the storeId for that item, then querying the DB for all the reviews associated with that storeId.

Reply:
Thanks for pointing it out, i didn't think of that, I was just thinking we are only showing one shop at this time, so I didn't create the shopID for the items, instead i just assigned every items belongs to the same shop. But you are right that I'll need to be more in line with REST and structure it better for further development, I'll work on that.